tensorboard/BUILD

```diff
@@ -436,6 +436,16 @@ py_library(
     visibility = ["//visibility:public"],
 )
 
+py_library(
+    name = "expect_pandas_installed",
+    # This is a dummy rule used as a pandas dependency in open-source.
+    # We expect pandas to already be installed on the system, e.g. via
+    # `pip install pandas`.
+    # NOTE: Unlike other parallel dependencies in this file, pandas is an
+    # optional dependency.
+    visibility = ["//visibility:public"],
```

Comment:
When you rebase master, please remove this `//visibility:public` spec,
as all the others were removed in #3568.

Reply:
Done.